### ISSUE TITLE
Multiple PDF Files

### ISSUE THREAD

**Issue by btxtiger** [restezconnectes/send-pdf-for-contact-form-7#44]
### Feature Request
I need an option to create multiple PDFs. 
In order to keep the current structure of the plugin, my suggestion is as follows:

- We could add a tag like `[addpage]`, but name it `[add_document]`. All content that comes after the tag, should be passed to the new document.
- The tag should have a suffix parameter to name the files: e.g. `[add_document suffix="-client_information"]` <br> => `data-client_information.pdf`
- Adding the tag in line 1 should set the suffix for the first file
- If I don't pass any suffix, and the template contains multiple `[add_document]`, the suffix should be automatically set to `-00`, `-01`, `-02`, ...

Would be very nice if we can add this feature. Let me know if I can help with it, however I am not yet familiar with the codebase.

**Comment by kirichme** [comment]
I support the user, it would be great)

**Comment by btxtiger** [comment]
Hi @restezconnectes ,

I wanted to follow up on this issue as it's been open for a while now, and the feature would be incredibly valuable to me (and perhaps others). I’d like to offer financial support to prioritize its development, if that would be helpful.

Please let me know if this is something you’d be interested in, or if there’s any other way I can assist in moving this forward. Thank you for your continued work!

Best regards

**Comment by restezconnectes** [comment]
Hi!

I'm working on it! i'm tryig to find the best solution...

Best regards

**Comment by btxtiger** [comment]
@restezconnectes thanks!
let me know if I can help with something

**Comment by restezconnectes** [comment]
Hi!

I've a new version with multi PDF generator. 
Take me your email for sending the zip file and testing this!

Thanks

**Comment by btxtiger** [comment]
@restezconnectes great news! Many thanks!
you can send it to btxtiger@icloud.com

**Comment by restezconnectes** [comment]
Hi!

Have you you tested y GitHub version: https://github.com/restezconnectes/send-pdf-for-contact-form-7/

Thanks for your report

**Comment by btxtiger** [comment]
Hey @restezconnectes ,
I'm really sorry for my late reply!

I tested the plugin today, and it looks really nice. The generating PDF works good.
The only thing that I noticed is, that the shortcodes dont work for the file name of the second PDF.

This is the name from the first PDF:
![Image](https://github.com/user-attachments/assets/96509728-d739-4c9a-a8f6-08dd67d5a822)

---
This is how i set the second:
![Image](https://github.com/user-attachments/assets/a383f532-d58f-4adc-8d68-9322a1244543)


---
This results in
![Image](https://github.com/user-attachments/assets/cfebe47b-2ac5-463b-97ad-262a8473a680)
instead of "widerrufsbelehrung-first_name-last_name.pdf"
